src/app/components/applications/SSV/ValidatorRegistrationConfirmation/ValidatorRegistrationConfirmation.tsx

```diff
@@ -103,23 +104,14 @@ const ValidatorRegistrationConfirmation = () => {
     {errorMessage && <ErrorMessage text={errorMessage}/>}
 
     <Grid container>
-      <Button
+      {termsConditionWrapper(<Button
```

Comment:
Use it as component which utilizes children usage with logic inside rather than current solution. It will be more react way